tests/utils.py

```diff
@@ -97,7 +97,7 @@ def formatError(self, test, err):
 
 		_, module, call = test.address()
 
-		output = ['python2', 'run_tests.py', u'%s:%s' % (module, call)]
+		output = ['python2', 'run_tests.py', u'{0}:{1}'.format(module, call)]
```

Comment:
weren't we going to use `{}` instead of `{0}`? You did use `{}` in you last Pull-Request.